src/pages/firstrun/StartServer.tsx

```diff
@@ -1,74 +1,60 @@
 import React from 'react';
-import { connect, ConnectedProps } from 'react-redux';
+import { useDispatch, useSelector } from 'react-redux';
 
 import { RootState } from '../../core/store';
 import Events from '../../core/events';
 import { setSaved as setFirstRunSaved } from '../../core/slices/firstrun';
 import Footer from './Footer';
-import Button from '../../components/Buttons/Button';
-
-class StartServer extends React.Component<Props> {
-  handleSave = () => {
-    const { setSaved, startServer } = this.props;
-    startServer();
-    setSaved('start-server');
-  };
-
-  handleNext = () => {
-    const { stopPollingStatus, user, signIn } = this.props;
-    stopPollingStatus();
-    signIn({
-      user: user.Username,
-      pass: user.Password,
-      device: 'web-ui',
-      redirect: false,
+import Button from '../../components/Input/Button';
+import TransitionDiv from '../../components/TransitionDiv';
+
+function StartServer() {
+  const dispatch = useDispatch();
+
+  const status = useSelector((state: RootState) => state.firstrun.serverStatus);
+  const user = useSelector((state: RootState) => state.firstrun.user);
+
+  const handleNext = () => {
+    dispatch({ type: Events.STOP_API_POLLING, payload: { type: 'server-status' } });
+
+    dispatch({
+      type: Events.AUTH_LOGIN,
+      payload: {
+        user: user.Username,
+        pass: user.Password,
+        device: 'web-ui',
```

Comment:
Maybe use different string for v2 ?

Reply:
Yeah, I could do it. But it shouldn't matter, right?